src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/VsProject_VsLangProjectProperties.cs

```diff
@@ -120,6 +120,28 @@ public string AbsoluteProjectDirectory
             }
         }
 
+        public bool AutoGenerateBindingRedirects
+        {
+            get
+            {
+                return _threadingService.ExecuteSynchronously(async () =>
+                {
+                    ConfigurationGeneralBrowseObject configurationGeneralProperties = await ProjectProperties.GetConfigurationGeneralBrowseObjectPropertiesAsync().ConfigureAwait(true);
+                    object value = await configurationGeneralProperties.AutoGenerateBindingRedirects.GetValueAsync().ConfigureAwait(true);
+                    return bool.Parse((string)value);
```

Comment:
I believe GetValueAsync here returns `bool?` for bool properties, so you'll want to cast this to (bool?) and not parse the result.

Reply:
Fixed.